toqito/matrices/cyclic_permutation.py

```diff
@@ -0,0 +1,36 @@
+"""Generate a cyclic permutation matrix."""
+
+import numpy as np
+
+
+def cyclic_permutation_matrix(n: int) -> np.ndarray:
+    r"""Create the cyclic permutation matrix P for a given dimension n.
```

Comment:
```suggestion
    r"""Create the cyclic permutation matrix P for a given dimension :code:`n`.
```
Does removing `r` from `r"""` perhaps solve the issue? I am not too sure about this.

Reply:
Didn't have to remove `r`. `:code: n`  fixed it.

Reply:
Something is buggy with this part. It was working, but not in the area that's expected.

Reply:
Issue was not importing the full function name in `matrices.rst`.